private/aws-middleware-test/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@aws-sdk/aws-middleware-test",
+  "description": "Integration test suite for AWS middleware",
+  "version": "3.0.0",
+  "scripts": {
+    "build": "exit 0",
+    "build:cjs": "exit 0",
+    "build:es": "exit 0",
+    "build:include:deps": "lerna run --scope $npm_package_name --include-dependencies build",
+    "build:types": "tsc -p tsconfig.types.json",
+    "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
+    "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
+    "test": "jest --coverage --passWithNoTests"
```

Comment:
these tests need to run in the `test:integration` classification, because they import undeclared fully built clients.